### PR TITLE
Fix defib mount runtime error.

### LINKED ISSUE #50376: Runtime on the PENLITE defibrillator mount
## Round ID: [133857](https://scrubby.melonmesa.com/round/133857)
 [01:43:10] Cannot read null.cell

Basically there isn't a defib in the mount.
Found the line of code that does it
````
if(defib?.cell?.charge < defib.cell.maxcharge && is_operational())
    use_power(100)
    defib.cell.give(80)
    update_icon()
````

? has a higher priority than < so the defib.cell.maxcharge causes the issue

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

Fixes #50376 

Incorrect use of var? functionality meant that a comparison was regularly firing on null vars.

Shifted said logic outside of the comparison with an early return.

**CODE UNTESTED AS THIS SHOULD BE A SIMPLE FIX WEBEDIT. PLEASE REVIEW THE CHANGES BEFORE MERGING. IT'S ONLY A COUPLE OF LINES OF CODE.**

## Why It's Good For The Game

Runtime bad.

## Changelog

No changelog necessary.

### CODE REVIEW

**Review by spookydonut — CHANGES_REQUESTED** [review]
This doesnt solve the problem of why its still processing without a defib in it or why cell-less defibs can be hooked up to charge

File: code/game/machinery/defibrillator_mount.dm [hunk 1]
```diff
@@ -159,12 +167,11 @@
 	wallframe_type = /obj/item/wallframe/defib_mount/charging
 
 /obj/machinery/defibrillator_mount/charging/process()
-	if(defib?.cell?.charge < defib.cell.maxcharge && is_operational())
+	if(defib.cell.charge < defib.cell.maxcharge && is_operational())
```
Comment (spookydonut) [inline code/game/machinery/defibrillator_mount.dm:170]: why not use get_cell()

**Comment by Timberpoes** [comment]
End processing before defib is nulled when defib is removed.
Start processing after defib is hooked up.

Guarded against cell-less defibs being hooked up.

Should actually address the underlying problems.

File: code/game/machinery/defibrillator_mount.dm (hunk 1 above)
Comment (spookydonut) [reply code/game/machinery/defibrillator_mount.dm:170]: var/obj/item/cell/C = get_cell()
if(C.charge < C.maxcharge && is_operation())

etc

Comment (Timberpoes) [reply code/game/machinery/defibrillator_mount.dm]: Changes should be made now.

**Review by spookydonut — APPROVED** [review]

**Pull request merged**